### PR TITLE
Clear angriness status when making guests happy, fixes #4505

### LINKED ISSUE #4505: The happiness cheat does not clear the 'red-faced with anger' status
example

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/cheats.c [hunk 1]
```diff
@@ -259,6 +259,12 @@ static void cheat_set_guest_parameter(int parameter, int value)
 		switch(parameter) {
 			case GUEST_PARAMETER_HAPPINESS:
 				peep->happiness = value;
+                // Clear the 'red-faced with anger' status if we're making the guest happy
+				if (value > 0)
+                {
```
Comment (LinusU) [inline src/cheats.c]: rest of the file uses tabs for indentation, looks strange here...

File: src/peep/peep.c [hunk 2]
```diff
@@ -5364,7 +5364,7 @@ static void peep_update_walking_break_scenery(rct_peep* peep){
 		(map_element->base_height << 3) + 32,
 		map_element->base_height << 3);
 
-	peep->var_F3 = 0x10;
+	peep->angriness = 0x10;
```
Comment (duncanspumpkin) [inline src/peep/peep.c]: 32.

**Comment by AaronVanGeffen** [comment]
Minor nit-pick: while in line with 'happiness', I think 'angriness' sounds a bit off. Perhaps we should just call it 'anger'?

File: src/peep/peep.c [hunk 3]
```diff
@@ -7297,7 +7297,7 @@ const int face_sprite_large[] = {
 static int get_face_sprite_offset(rct_peep *peep){
 
 	// ANGRY
-	if (peep->var_F3) return PEEP_FACE_OFFSET_ANGRY;
+	if (peep->angriness) return PEEP_FACE_OFFSET_ANGRY;
```
Comment (IntelOrca) [inline src/peep/peep.c]: != 0

**Comment by Gymnasiast** [comment]
OK, made the requested changes.

**Pull request merged**